components/supported/SupportedDefi.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div id="defi" class="py-12">
+    <heading>Dedicated DeFi support for</heading>
+    <div class="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-4 gap-4 mt-8">
+      <supported-icon-wrapper
+        v-for="item in defi"
+        :key="item.name"
+        :name="item.name"
+        :img="item.img"
+      />
+    </div>
+    <Caption
+      >* All DeFi protocols are supported, these are just the ones we have
```

Comment:
Can we maybe either extract that to en.json or add spaces so that the closing tag doesn't break?